MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Defaults.xaml

```diff
@@ -14,6 +14,7 @@
     <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.DatePicker.xaml" />
     <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.DialogHost.xaml" />
     <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Expander.xaml" />
+    <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.FocusVisual.xaml" />
```

Comment:
Shouldn't this addition also be added to the `MaterialDesign3.Defaults.xaml` for it to have any effect on the material design 3 styles (there is a separate demo app where you can test this)?

Probably also to `MaterialDesign2.Defaults.xaml` which I think is supposed to eventually replace `MaterialDesignTheme.Dafaults.xaml`. @Keboo correct me if I am wrong...

Reply:
Added